docs/topics/gradle/gradle-configure-project.md

```diff
@@ -81,7 +81,8 @@ methods and properties, and supports all the current Gradle features.
 By default, the Kotlin Gradle plugin stores persistent project-specific data at the root of the project,
 in the `.kotlin` directory.
 
-> You may want to add the `.kotlin` directory to your project's `.gitignore` file.
+> Do not commit the `.kotlin` directory to version control.
```

Comment:
Is there a reason why? It would be nice if we could explain with a couple of words. But just a suggestion, up to you.

Reply:
as discussed it would be better to keep it as unambiguous as possible